mycli/main.py

```diff
@@ -111,6 +114,14 @@ def __init__(self, sqlexecute=None, prompt=None,
         # Register custom special commands.
         self.register_special_commands()
 
+        # Load .mylogin.cnf if it exists.
+        mylogin_cnf_path = get_mylogin_cnf_path()
+        if mylogin_cnf_path:
+            mylogin_cnf = get_mylogin_cnf_plaintext(mylogin_cnf_path)
```

Comment:
I think it should continue. It can print an error message but ignore that file and continue. That's the current behavior with malformed `my.cnf` files.